apps/wpcom-block-editor/src/wpcom/features/live-preview/hooks/use-previewing-theme.ts

```diff
@@ -22,11 +26,7 @@ const getThemeType = ( theme?: Theme ) => {
 	}
 
 	// @TODO Replace all the logic above with the following code once Theme Tiers is live.
-	if ( config.isEnabled( 'themes/tiers' ) ) {
-		return theme?.theme_tier?.slug ?? undefined;
-	}
-
-	return undefined;
+	return theme?.theme_tier?.slug ?? undefined;
```

Comment:
I think it's safe to remove the feature checks on the Live Preview (we can't enable them with flags) since you can only open the Live Preview of a Personal theme if you're using the Theme Showcase with the flag enabled.

Reply:
When I preview a free theme it says "a undefined theme"
<img width="361" alt="Screenshot 2023-12-22 at 15 20 10" src="https://github.com/Automattic/wp-calypso/assets/93301/7244bb34-0a62-4147-9a97-7e69b11762aa">

Reply:
That shouldn't be related to the feature check. I'll have a look at what's going on with the free tier.

Reply:
I can't reproduce this, it's working fine for me on free themes. How did you get Pixl to live preview? AFAIK Pixl doesn't qualify for Live Preview.

Reply:
That nudge doesn't appear for me now. I think the live preview UI has changed. I just get this banner instead:
<img width="1165" alt="Screenshot 2024-01-09 at 13 55 03" src="https://github.com/Automattic/wp-calypso/assets/93301/3a551ca7-32a3-446b-a2a1-6fef883fc567">

FWIW I used the "Preview & Customize" button here to test:
<img width="716" alt="Screenshot 2024-01-09 at 13 53 59" src="https://github.com/Automattic/wp-calypso/assets/93301/c3357a3e-517a-4ac3-a3ed-416477122b7f">